generators/client/templates/angular/src/main/webapp/app/shared/_shared-common.module.ts

```diff
@@ -54,7 +54,11 @@ import {
         Title,
         {
             provide: LOCALE_ID,
+        <%_ if (nativeLanguage === 'ar-ly') { _%>
+            useValue: 'en'
```

Comment:
@deepu105 anyway, this line `useValue: 'en'` will never care about changing the language on run-time.
You can try to use `useValue: 'ar-ly'` in one of your apps after that even if you change to any other language at run-time it will always show Arabic numbers especially Dates you can check Created date at User management.